spec/controllers/home_controller_spec.rb

```diff
@@ -4,9 +4,60 @@
   render_views
 
   describe 'GET #index' do
-    it 'redirects to about page' do
-      get :index
-      expect(response).to redirect_to(about_path)
+    context 'when not signed in' do
+      it 'redirects to about page' do
+        get :index
+        expect(response).to redirect_to(about_path)
+      end
+    end
+
+    context 'when signed in' do
+      let(:user) { Fabricate(:user) }
+      subject do
+        sign_in(user)
+        get :index
+      end
+
+      it 'assigns @body_classes' do
+        subject
+        expect(assigns(:body_classes)).to eq 'app-body'
```

Comment:
Yeah, that's fine -- just wanted to make note of it.

Similarly - I think there's a bunch of stuff in controller specs (a lot of which I've personally added) which would be better placed in either request or feature specs.

I've been prioritizing hitting C0 coverage first, and then ongoing refactors can come after that.

I have no objection to continuing to merge these things.